cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -403,9 +406,12 @@ class fixed_point_scalar : public scalar {
     rmm::cuda_stream_view stream = cudf::get_default_stream()) const;
 
   /**
-   * @brief Explicit conversion operator to get the value of the scalar on the host.
+   * @brief Returns the value of the scalar as decimal32, decimal64 or decimal128 on the host.
+   *
+   * @param stream CUDA stream used for device memory operations.
+   * @return The value of the scalar
    */
-  explicit operator value_type() const;
+  T get_value(rmm::cuda_stream_view stream = cudf::get_default_stream()) const;
```

Comment:
Maybe we should delete this. There are already `value` and `fixed_point_value` functions.